src/Data/Nat/Binary/Minus.agda

```diff
@@ -0,0 +1,364 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Subtraction on Bin and some of its property proofs.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Nat.Binary.Minus where
+
+open import Algebra using (Magma)
+import Algebra.FunctionProperties as FuncProp
+open import Data.Empty using (⊥)
+open import Data.Fin using (#_)
+open import Data.Nat as ℕ using (ℕ)
+open import Data.Nat.Binary.Base
+open import Data.Nat.Binary.Properties
+import Data.Nat.Properties as ℕₚ
+open import Data.Product using (_×_; _,_; proj₁; proj₂; ∃)
+open import Data.Sum using (inj₁; inj₂)
+open import Data.Vec using ([]; _∷_)
+open import Function using (_∘_; _$_)
+open import Level using (0ℓ)
+open import Relation.Binary using (Tri; _Preserves_⟶_; _Preserves₂_⟶_⟶_)
+open import Relation.Binary.PropositionalEquality as P using
+  (_≡_; _≢_; _≗_; refl; sym; trans; cong; cong₂; resp₂; subst; magma;
+   module ≡-Reasoning)
+open import Relation.Nullary using (Dec; yes; no)
+open import Relation.Nullary.Negation using (contradiction)
+open import Algebra.FunctionProperties {A = ℕᵇ} _≡_
+open import Algebra.Properties.CommutativeSemigroup +-commutativeSemigroup
+  using (xy∙z≈y∙xz; x∙yz≈y∙xz)
+open import Algebra.Solver.CommutativeMonoid +-0-commutativeMonoid
+
+private module FP-Bin = FuncProp (_≡_ {A = ℕᵇ})
```

Comment:
Why not just open this directly at the top of the file rather than creating a new module? In fact you are already doing this, so I don't think you need this module at all.

Reply:
Fixed. It remains only  ``open import Algebra.Definitions {A = ℕᵇ} _≡_``                                    
at the top.